packages/@misk/cli/src/utils/generate.ts

```diff
@@ -48,11 +48,11 @@ export const generateBuildFiles = async (...args: any) => {
     ),
     fs.writeFile(makePath(dir, Files.gitignore), createGitignore(miskTab)),
     fs.writeFile(makePath(dir, Files.webpack), webpack)
-  ]
+  ])
 
   // Append non-package JSON files with generated by comment
-  fs.writeFile(makePath(dir, Files.tsconfig), generatedByCLI, { flag: "a" })
-  fs.writeFile(makePath(dir, Files.tslint), generatedByCLI, { flag: "a" })
+  await fs.writeFile(makePath(dir, Files.tsconfig), generatedByCLI, { flag: "a" })
+  await fs.writeFile(makePath(dir, Files.tslint), generatedByCLI, { flag: "a" })
```

Comment:
These are the async versions of these functions, so I think they should be awaited on.